### ISSUE TITLE
Scan API request format is broken

### ISSUE THREAD

**Issue by cbisnett** [picatz/shodanz#48]
I recently upgraded from 2.0.0 to 2.0.3 and found that `client.scan` no longer works. I dug in a bit, but I wasn't able to figure out what exactly was wrong with the request, but something about the format or the params is broken. I've verified with the Python Shodan library, that scanning still works as well as `curl` and I used the same API key in all cases. It looks like this could be an issue that was introduced when changing from `Async::HTTP::Internet` to `Async::HTTP::Client`, but the documentation is scant for those libraries and the source code didn't give any hints either. Below is the error response I get when trying to request a scan (`$shodan` is an instance of `Client`):

```ruby
Portal (DEVELOPMENT) (main) 001:0> $shodan.scan "1.1.1.1"
JSON::ParserError: 784: unexpected token at '<html>
<head><title>502 Bad Gateway</title></head>
<body bgcolor="white">
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx</center>
</body>
</html>
'
from /home/chris/.rbenv/versions/2.5.5/lib/ruby/gems/2.5.0/gems/json-2.3.1/lib/json/common.rb:263:in `parse'
```

**Comment by picatz** [comment]
Thanks for reporting this @cbisnett, I'll work on a fix.

**Comment by picatz** [comment]
Fix now available in [v2.0.4](https://github.com/picatz/shodanz/releases/tag/v2.0.4)
